rpc_jobs/pull_request_whisperer.yml

```diff
@@ -0,0 +1,51 @@
+- job-template:
+    name: 'Pull-Request-Whisperer_{repo}'
+    project-type: workflow
+    concurrent: true
+    run_commenter: true
+    run_issue_linker: false
+    comment_body: |
+      Thank you for submitting a pull request to this project!
+
+      Once all `check` jobs have passed and the pull request has received a
+      sufficient number of reviews, the repository's `gate` jobs can be triggered by
+      adding a comment of `check_component_gate` to this pull request.
```

Comment:
I think this will trigger the gate job. `trigger-phrase: '(re)?check_component_gate'`

Reply:
@hughsaunders the code appears to [match not find](https://github.com/jenkinsci/ghprb-plugin/blob/4e86ed47a96a01eeaa51a479ff604252109635f6/src/main/java/org/jenkinsci/plugins/ghprb/Ghprb.java#L102) and so won't trigger the job. I've tested this to confirm.